homeassistant/components/asuswrt.py

```diff
@@ -37,7 +39,9 @@
         vol.Optional(CONF_REQUIRE_IP, default=True): cv.boolean,
         vol.Exclusive(CONF_PASSWORD, SECRET_GROUP): cv.string,
         vol.Exclusive(CONF_SSH_KEY, SECRET_GROUP): cv.isfile,
-        vol.Exclusive(CONF_PUB_KEY, SECRET_GROUP): cv.isfile
+        vol.Exclusive(CONF_PUB_KEY, SECRET_GROUP): cv.isfile,
+        vol.Optional(CONF_SENSORS, default=None): vol.All(
```

Comment:
Remove the default and use `dict.get` when getting this config key.

Reply:
Sorry about that...  And thank you for spotting it!